CHANGES.md

```diff
@@ -67,6 +67,11 @@
 
 ## Breaking Changes
 
+* The Java artifacts "beam-sdks-java-io-kinesis", "beam-sdks-java-io-google-cloud-platform", and
+  "beam-sdks-java-extensions-sql-zetasql" declare Guava 30.1-jre dependency (It was 25.1-jre in Beam 2.27.0).
+  This new Guava version may introduce dependency conflicts if your project or dependencies rely
+  on removed APIs. If affected, ensure to use an appropriate Guava version via `dependencyManagement` in Maven and
+  `force` in Gradle.
```

Comment:
@aaltay I added this note for potential impact to Beam users. The potential risk described here is not special to this Guava version. Every dependency upgrade, in general, carries a risk of introducing dependency conflicts if a user relies on removed methods or classes. (Therefore this note might not be needed.)